akka-actor-testkit-typed/src/main/scala/akka/actor/testkit/typed/internal/TestProbeImpl.scala

```diff
@@ -118,18 +123,19 @@ private[akka] final class TestProbeImpl[M](name: String, system: ActorSystem[_])
   def within[T](max: JDuration)(f: Supplier[T]): T =
     within_internal(Duration.Zero, max.asScala.dilated, f.get())
 
-  private def within_internal[T](min: FiniteDuration, max: FiniteDuration, f: ⇒ T): T = {
+  private def within_internal[T](min: FiniteDuration, max: FiniteDuration, f: => T): T = {
     val start = now
     val rem = if (end == Duration.Undefined) Duration.Inf else end - start
     assert(rem >= min, s"required min time $min not possible, only ${rem.pretty} left")
 
     lastWasNoMessage = false
 
-    val maxDiff = max min rem
+    val maxDiff = max.min(rem)
     val prevEnd = end
     end = start + maxDiff
 
-    val ret = try f finally end = prevEnd
+    val ret = try f
+    finally end = prevEnd
```

Comment:
Weird formatting here

Reply:
Let's not do that. Something with `Term`?
https://github.com/scalameta/scalafmt/pull/1187

Reply:
couldn't find a cure for this one

Reply:
Yeah, long-standing bug.